### ISSUE TITLE
Gaps in some sessions

### ISSUE THREAD

**Issue by moorepants** [mechmotum/baby-vibration#16]
I noticed these gaps in the session bike speed plots:

![image](https://github.com/user-attachments/assets/5a421676-3760-4f51-beab-c9852ede188d)
![image](https://github.com/user-attachments/assets/f4400e2a-74c8-4700-afcc-21fb9c34bb31)

Not sure why they are there. Need to investigate.

@GabrieleDello any offhand idea of why this would be?

**Comment by GabrieleDello** [comment]
Nothing remarkable happened. The only guess I can make is that we wrongly pushed the button to start/stop the acquisition two times. So,basically, we can state as follows:
- top plot (session005): the gap is at the beginning of acquisition, when we turn on all the sensors. Not sure of the start of the acquisition, we pressed again (stop a running acquisition), held on for a few seconds, and pushed the button again since we saw the sensor blinking green again.
- bottom plot (session016): the gap is in the end. After the test run, we stopped and turned off all the sensors (which actually happened and it is visible at about 13 11:40). Then, back to the lab, we might have pushed the button again (perhaps accidentally while getting the sensors off the bike) to realize that when I reconnected the sensor to the Shimmer base (which automatically stopped the acquisition).
